_rules/auto-update-text-efbfc7.md

```diff
@@ -43,7 +43,7 @@ For each test target there exists an [instrument][] to achieve one of the follow
 
 ## Expectation 2
 
-The [instrument][] from Expectation 1 is in one of the following:
+The set of [instruments][instrument] from Expectation 1 is in one of the following:
```

Comment:
Do you mean that all instruments in the set are in the same place, or can they be in different places?

What if the "pause" button is on the web page but the "resume" button is on a modal?